boards/arc/em_starterkit/doc/board.rst

```diff
@@ -123,6 +132,26 @@ pieces of hardware are required.
   See http://store.digilentinc.com/pmod-modules or develop your
   custom interfaces to attach to the Pmod connector.
 
+Set up the ARC EM Starter Kit
+=============================
+
+To run Zephyr application on correct arc core of EM Starter Kit, you need to
+setup the board correctly.
+
+* Connect the digilent usb cable from your host to the board.
+
+* Connect the 5V DC power supply to your board.
+
+* Select the core configuration of the board by choosing correct dip switch
+  SW1 settings, then press then FPGA configure button located above the letter
+  'C' of the ARC logo on the board.
+
+* Then the board will be reconfigured with selected core configuration, you
+  can download and debug Zephyr application now.
+
+* If you want to know more about how to use this board, you can take a look
+  at the **ARC EM Starter Kit User Guide**.
```

Comment:
OK, I will add it, I thought the document version hosted on https://www.embarc.org/help.html#starterkit was an old one for EM StarterKit, but now it seems it is updated to latest version. So I will add link to it.